client/src/app/modules/app-form/config/validation.json

```diff
@@ -17,10 +17,11 @@
   },
   "quickstarters": {
     "componentName": {
-      "regex": "(([A-Za-z][-A-Za-z]*)?[A-Za-z]){3,40}",
+      "regex": "^(?!tests)(([A-Za-z][-A-Za-z]*)?[A-Za-z]){3,40}",
       "errorMessages": [
         "Please enter 3-40 characters",
-        "Please use letters and dashes (optional)"
+        "Please use letters and dashes (optional)",
+        "Please avoid using reserved keywords e.g. tests"
```

Comment:
Either the message should be changed to `Please avoid using tests as a reserved keyword` or the regex should be really extended with a **list of keywords**, then the message makes sense again.

Reply:
It makes total sense - thanks for the feedback @netzartist